tests/docker_run_tests.rs

```diff
@@ -34,55 +36,68 @@ fn remove_containers(container_id: &str) {
         .arg("rm")
         .arg(container_id)
         .spawn()
+        .unwrap()
+        .wait()
+        .context("Removing container")
         .unwrap();
 }
 
-fn stop_and_remove_container(image: String) {
+fn stop_and_remove_container_by_image(image: String) {
     let container_ids = get_container_ids_from_image(image);
     let container_id = container_ids.trim().split('\n').collect::<Vec<_>>()[0].to_string();
 
-    stop_containers(&container_id);
-    remove_containers(&container_id);
+    stop_and_remove_container(container_id);
+}
+
+fn stop_and_remove_container(name: String) {
+    stop_containers(&name);
+    remove_containers(&name);
 }
 
+struct Config {
```

Comment:
Nit: rename to `DockerRunConfig`

Reply:
It's not techincally a run config cause it's also passed to and from run

Open to other names though it's a bit generic